django/db/models/base.py

```diff
@@ -298,6 +298,12 @@ def __new__(cls, name, bases, attrs):
                 else:
                     new_class.add_to_class(field.name, copy.deepcopy(field))
 
+        # Set name for _meta.indexes. We cannot do this in Options.contribute_to_class
```

Comment:
Set the name of _meta.indexes. This can't be done in Options.contribute_to_class() because fields haven't been added to the model yet.